index.html

```diff
@@ -1221,6 +1221,10 @@ <h3>Co-signed</h3>
               <td>Company</td>
               <td>Open-source community efforts</td>
             </tr>
```

Comment:
It seems that it need a "<tr>" in the line 1224.

Reply:
Updated, thanks!